src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/services/v2/AuthorizationStateRetrieverService.kt

```diff
@@ -0,0 +1,46 @@
+package it.pagopa.ecommerce.eventdispatcher.services.v2
+
+import it.pagopa.ecommerce.commons.client.NpgClient
+import it.pagopa.ecommerce.commons.domain.TransactionId
+import it.pagopa.ecommerce.commons.exceptions.NpgResponseException
+import it.pagopa.ecommerce.commons.generated.npg.v1.dto.StateResponseDto
+import it.pagopa.ecommerce.commons.utils.NpgPspApiKeysConfig
+import it.pagopa.ecommerce.eventdispatcher.exceptions.NpgBadRequestException
+import it.pagopa.ecommerce.eventdispatcher.exceptions.NpgServerErrorException
+import java.util.*
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.stereotype.Service
+import reactor.core.publisher.Mono
+
+@Service
+class AuthorizationStateRetrieverService(
```

Comment:
is this class a duplicate of this [one](https://github.com/pagopa/pagopa-ecommerce-event-dispatcher-service/blob/main/src/main/kotlin/it/pagopa/ecommerce/eventdispatcher/services/v2/NpgStateService.kt)?
Code seems to be duplicated

Reply:
That's right. A rebase from main did resurrect it